cmake/modules/extensions.cmake

```diff
@@ -4801,9 +4801,13 @@ endmacro()
 # Output and source files must be specified using the OUTPUT and SOURCES
 # arguments. Only one source file is currently supported.
 #
-# Arch-specific flags will be added automatically. The C_FLAGS argument
-# can be used to pass additional compiler flags to the compilation of
-# the source file.
+# The llext code will be compiled with mostly the same C compiler flags used
+# in the Zephyr build, but with some important modifications. The list of
+# flags to remove and flags to append is controlled respectively by the
+# LLEXT_REMOVE_FLAGS and LLEXT_APPEND_FLAGS global variables.
+
+# The C_FLAGS argument can be used to pass additional compiler flags to the
+# compilation of this particular llext.
```

Comment:
in CI we use _command-line_ parameters like `-DEXTRA_CXXFLAGS=Werror` a lot. Will they work with `llext` too?

Reply:
I am no CMake guru so I just tested - it does (well, with EXTRA_CFLAGS as it's a C file, but it does indeed use command line flags in the llext as well).